### PR TITLE
Recommend download chrome in inferior browser

### PR DESCRIPTION
**Pull request by smallst**
test in Edge, 360 browser, QQ browser, Sougou browser
as their webkit core throw error with _.js

### CODE REVIEW

**Comment by smallst** [comment]
recommendation appears in the top bar of frontpage and alert modal in play page

File: app/views/core/RootView.coffee [hunk 1]
```diff
@@ -201,10 +201,11 @@ module.exports = class RootView extends CocoView
       #console.log 'Saved language:', newLang
 
   isOldBrowser: ->
-    if $.browser
+    if features.china and $.browser
```
Comment (wjllance) [inline app/views/core/RootView.coffee]: Do we use this function for other area?

Comment (smallst) [reply app/views/core/RootView.coffee]: nick says no

Comment (smallst) [reply app/views/core/RootView.coffee]: and i serach through source code and find none

**Pull request merged**